.github/workflows/valid-image-generation-test.yml

```diff
@@ -47,9 +48,10 @@ jobs:
 
       - name: Run Docker container
         run: |
-          (timeout 30s docker run --env-file "$(pwd)/.env" $REGISTRY/$DOCKER_IMAGE_NAME) ||
+          (timeout 60s docker run --env-file "$(pwd)/.env" $REGISTRY/$DOCKER_IMAGE_NAME) ||
           if [ $? -eq 124 ]; then
             echo "OK, timeout error is acceptable."
           else
             exit $?
           fi
+        continue-on-error: true
```

Comment:
Does not smells fine

Reply:
It's inherit from previous PR https://github.com/GeoSales-Evolution/multi-tenant-uploader/pull/54

Reply:
Already mapped in https://github.com/GeoSales-Evolution/multi-tenant-uploader/issues/57